pwa/src/components/TreatmentForm.test.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import TreatmentForm from './TreatmentForm'
+import * as data from './TreatmentFields.json'
+
+/*
+
+Treatment Fields are stored in TreatmentFields.json in the format:
+
+{
+    "treatments" : [
+        {
+            "id": "multivitamins",
+            "group-title": "Multivitamins",
+            "group": [
+                { "name": "Multivitamins - Childrens", "id": "vitamin-childrens" },
+                { "name": "Multivitamins - Adult", "id": "vitamin-adult" },
+                { "name": "Multivitamins - Prenatal", "id": "vitamin-prenatal" }
+            ]
+        },
+        {...}
+    ]
+}
+*/
```

Comment:
Is the inclusion of this comment intentional?

Future developers could intuit this from the import from `TreatmentFields.json`

Reply:
Yes, it was more of a convenience comment block. Without having to cross-reference files they can get a quick idea of the structure of the JSON file where it was immediately relevant.

Reply:
I left similar comments in #84 , if it is removed here we should remove those also (alternately, keep both). I agree with both comments; the format could easily be inferred from the JSON file, but it was added for convenience when writing the component/test.

Reply:
I'd vote to keep it included for now. There are some objects in the JSON that are the single example of an object property, and they're in the middle of the array of objects. So having one guaranteed object with all possible parameters has been helpful for me personally.